### ISSUE TITLE
@osdi scope on NPM

### ISSUE THREAD

**Issue by natemoo-re** [opensupporter/osdi-docs#329]
Related to #328. In the process of finding some way to publish my TypeScript type definitions for OSDI, I discovered that the `@osdi` scope on NPM was still available. 

I created an organization at http://npmjs.com/org/osdi and would be happy to invite anyone to the organization, or even turn it over to the committee. 

It could be a great place to publish official opensupporter resources for the JavaScript/TypeScript communities. For example, `npm install @osdi/types` could provide developers with the types for every data structure in the spec and down the line the jQuery plugin could be replaced with something like `@osdi/client` (which is another project I’d love to tackle!)

**Comment by joshco** [comment]
We're not really node experts, so we don't understand what this means. It sounds like it would be useful for the node community.  Would you be able to attend a future call to educate us on what this is and how it would be used?

**Comment by natemoo-re** [comment]
Of course, I totally understand that. I'd be happy to attend a call and discuss it!

**Comment by joshco** [comment]
Nate, can you email me at info@opensupporter.org ?  we have a meeting tomorrow.

**Comment by joshco** [comment]
@natemoo-re can you email me at info@opensupporter.org ? we have a meeting tomorrow.

**Comment by joshco** [comment]
@natemoo-re If there's no further activity on this, the issue will be closed. 
If you are interested in providing technical feedback and wish to do so privately, in the closet, you can contact me at the OSDI Closet Hotline (516) 701 1900. You feedback will be related anonymously and your identity will only be known by me.

**Comment by natemoo-re** [comment]
Thanks @joshco – unfortunately I don't have the bandwidth to move forward on this right now. Should that change, I will get in touch.

**Comment by joshco** [comment]
People have reported the concurrence of homophobic bullying behaviors like behind the back gossip where their target's voice has been silenced, as well as being threatened with retaliation if they don't participate in a homophobic boycott.  If you've have been or come to be exposed to this behavior, and believe the rights of all minorities should be respected on an equal footing, that their voices should be heard, and you wish to be fully informed of the LGBT rights side of this conflict, feel free contact me.
